src/Cli/dotnet/SlnFileFactory.cs

```diff
@@ -78,7 +78,7 @@ public static SolutionModel CreateFromFilteredSolutionFile(string filteredSoluti
             try
             {
                 JsonElement root = JsonDocument.Parse(File.ReadAllText(filteredSolutionPath)).RootElement;
-                originalSolutionPath = root.GetProperty("solution").GetProperty("path").GetString();
+                originalSolutionPath = Uri.UnescapeDataString(root.GetProperty("solution").GetProperty("path").GetString());
                 filteredSolutionProjectPaths = root.GetProperty("solution").GetProperty("projects").EnumerateArray().Select(p => p.GetString()).ToArray();
```

Comment:
Should we also unescape the absolute path and project paths?

Reply:
Yes, I thought about unescaping the absolute path, but that is generated by the system so I wouldn't expect it to be escaped (as it would by the json serializer), and maybe it tries to incorrectly parse certain characters. I may be completely wrong tho, would love to hear opinions on this. 
For the project paths, you're right, I agree